internal/restinterface/route/route_test.go

```diff
@@ -150,3 +138,36 @@ func TestStartFakeRoute(t *testing.T) {
 	}
 	router.Start()
 }
+
+func TestStartSecureRoute(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	router := NewRestRouterWithCerti(fakeCertsPath)
+	if router == nil {
+		t.Error(unexpectedFail)
+		return
+	}
+	router.Start()
+}
+
+// func TestStartFakeRoute(t *testing.T) {
```

Comment:
Do we need to leave this deactivated source codes specifically?

Reply:
Dear @MoonkiHong, thank you for following the project. I'm trying to understand why the port is not being closed on the test server. Locally everything works for me, so I'm looking for a problem on the server, sorry for the inconvenience. When there is a final decision, I will write to you to make a review.